src/sage/topology/simplicial_complex.py

```diff
@@ -4802,7 +4802,85 @@ def intersection(self, other):
             F = F + [s for s in self.faces()[k] if s in other.faces()[k]]
         return SimplicialComplex(F)
 
+    @cached_method
+    def bigraded_betti_numbers(self):
+        r"""
+        Returns a dictionary of the bigraded Betti numbers of ''self'', 
+        with keys '(-i, 2j)'.
+
+        See bigraded_betti_number(self, a, b) for more information.
+
+        EXAMPLES::
 
+            sage: X = SimplicialComplex([[0,1],[1,2],[1,3],[2,3]])
+            sage: Y = SimplicialComplex([[1,2,3],[1,2,4],[3,5],[4,5]])
+            sage: X.bigraded_betti_numbers()
+            {(0, 0): 1, (-1,4 ): 2, (-1, 6): 1, (-2, 6): 1, (-2, 8): 1}
+            sage: Y.bigraded_betti_numbers()
+            {(0, 0): 1, (-1,4 ): 3, (-2, 6): 1, (-2, 8): 2, (-3, 10): 1}
+        """
+        from sage.misc.functional import gens 
+        from sage.homology.homology_group import HomologyGroup
+        L = set(self.vertices())
+        n = len(L)
+        B = {}
+        H0 = HomologyGroup(0, ZZ)
+
+        for i in range(n+1):
+            for j in range (n+1):
+                B[(-i, 2*j)] = 0
+
+        B[(0,0)] = 1
+
+        for j in range(n+1):
+            for x in combinations(L, j):
+                S = self.generated_subcomplex(x)
+                H = S.homology()
+                for k in range(j):
+                    if H.get(j-k-1) != H0 and H.get(j-k-1) is not None:
+                        B[(-k, 2*j)] += len(H.get(j-k-1).gens())
+	    
+        return B
+
+    @cached_method
+    def bigraded_betti_number(self, a, b):
+        r"""
+        Returns the bigraded Betti number indexed in the form (-i, 2j).
+
+        Bigraded Betti number with indices '(-i, 2j)' is defined as a sum of ranks
+        of '(i-j-1)'-th (co)homologies of full subcomplexes with exactly 'j' vertices. 
```

Comment:
```suggestion
        Bigraded Betti number with indices `(-i, 2j)` is defined as a sum of ranks
        of `(i-j-1)`-th (co)homologies of full subcomplexes with exactly `j` vertices. 
```

Reply:
Yeah, I misread those backticks as single quotes. Sorry, and thank you.